solr/config/configsets/all_collection_oag56_v1/managed-schema

```diff
@@ -526,11 +506,8 @@
   <field name="helpdesk_email" type="string" docValues="true" indexed="true" stored="false" useDocValuesAsStored="true"/>
   <field name="helpdesk_url" type="text_general" sortMissingLast="true" required="false"/>
   <field name="horizontal" type="boolean" uninvertible="true" docValues="true" indexed="true" stored="true"/>
-  <field name="hosting_legal_entity" type="text_general"/>
   <field name="id" type="string" termPositions="true" termVectors="true" termOffsets="true" required="true" useDocValuesAsStored="true"/>
-  <field name="iid" type="plong" required="false"/>
```

Comment:
This should be in the schema

Reply:
Yes, you're totally right I made a mistake by using an outdated schema. In new schema, this field is available.